RELEASES.md

```diff
@@ -370,5 +370,17 @@ the total stake. The remaining 33% of the stake is configured to belong to
 a validator that is never actually run in the test network. The network is run
 for multiple days, ensuring that it is able to produce blocks without issue.
 
+#### Vote-extension Testnet
```

Comment:
Can we move this one before `Network Partition Testnet` (so we leave those never run at the end)?